config/initializers/marshal_autoloader.rb

```diff
@@ -1,6 +1,6 @@
-module Marshal
-  def self.load_with_autoload_missing_constants(data)
-    load_without_autoload_missing_constants(data)
+module MarshalMissing
```

Comment:
Can you name it MarshalAutoloader to match the file name? (or just rename the file, either one)